translations/bnpchs-final-2018-blusukan.html

```diff
@@ -0,0 +1,95 @@
+<!-- Title: Blusukan -->
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Bollyjee presidential election is around the corner. Jelix as the presidential 
+  candidate wants to realize the national commitments by doing <i>blusukan</i> (blusukan 
+  is indonesian word that refer to hands-on interaction with public in order to notice their 
+  necessity).
+</p>
+
+<p style="text-align: justify;">
+  There are a bunch of islands and $N$ province in Bollyjee. The island of two provinces lie on could be the 
+  same or different. Jelix had $M$ information, each information provide two provinces 
+  that lie on same island. If province $A$ and $B$ lie on the same island and province $B$ and $C$ lie 
+  on the same island then province $A$ and $C$ also lie on the same island.
+</p>
+
+<p style="text-align: justify;">
+  Each pair of province in the island is connected by a road. Also for each pair of road 
+  there are pavement that connect them so that Jelix can go from a road to other road with this 
```

Comment:
```suggestion
  there is an alley that connects them so that Jelix can go from a road to the other road through this 
```

hmm "pavement" kayanya kurang tepat, karena itu artinya trotoar untuk pejalan kaki (dan biasanya ada di sepanjang jalan), bukan penghubung antar jalan
kayanya pakai "alley" aja, jadinya semuanya diganti